Phaser/phaser-workshop.adoc

```diff
@@ -235,10 +248,13 @@ Voeg onderstaande code toe aan het einde van de functie 'sterPakken' zodat er me
 ----
 // wat moet er gebeuren als alle sterren zijn verzameld?
 if (sterren.countActive(true) === 0) {
+
+    // Zet alle sterren weer aan
```

Comment:
Ik zou zelf zeggen: // maak weer een nieuwe groep sterren. [het is immers niet zo dat de sterren nu als gloeilampjes uit zijn gegaan]